tryfirst/Dockerfile

```diff
@@ -1,9 +1,11 @@
 FROM kytos/base
 
 RUN python3.6 -m pip install --upgrade pip setuptools wheel
-RUN python3.6 -m pip install kytos==2019.1b1
+RUN python3.6 -m pip install python-openflow==2019.1b3
+RUN python3.6 -m pip install kytos-utils==2019.1b3
+RUN python3.6 -m pip install kytos==2019.1b3
```

Comment:
I think pip has a parameter to install the latest beta release. Please change this so we don't need to version bump here everytime.